src/test/kotlin/sk/streetofcode/webapi/integration/CourseIntegrationTests.kt

```diff
@@ -109,11 +111,12 @@ class CourseIntegrationTests : IntegrationTests() {
             fetchedCourse.thumbnailUrl shouldBe "editedThumbnailUrl"
             fetchedCourse.iconUrl shouldBe "editedIconUrl"
             fetchedCourse.status shouldBe CourseStatus.PRIVATE
+            fetchedCourse.courseOrder shouldBe 2
         }
 
         "delete course" {
             val uniqueSlug = getRandomString()
-            val course = addCourse(CourseAddRequest(1, 1, "testName", uniqueSlug, "short", "long", null, "trailerUrl", "trailerUrl", "thumbnailUrl", CourseStatus.PRIVATE))
+            val course = addCourse(CourseAddRequest(1, 1, "testName", uniqueSlug, "short", "long", null, "trailerUrl", "trailerUrl", "thumbnailUrl", CourseStatus.PRIVATE, 1))

```

Comment:
I don't think it matters too much so I won't insist on it, but maybe it would be worth it to add a test that the courses are indeed sorted by the `courseOrder` 🤷

Reply:
I will add test